ghost/members-api/lib/repositories/member.js

```diff
@@ -932,6 +938,16 @@ module.exports = class MemberRepository {
                 mrr_delta: model.get('mrr'),
                 ...eventData
             }, options);
+
+            // Notify paid member subscription start
+            if (this._labsService.isSet('emailAlerts')) {
+                await this.staffService.notifyPaidSubscriptionStart({
+                    member: member.toJSON(),
+                    offer: offer ? this._offerRepository.toJSON(offer) : null,
```

Comment:
It would be preferable to either use the `Offer` object here OR if you want a POJO, use the offer service rather than repository - it's not a massive deal as we're gonna reshuffle stuff around

Reply:
Yeah so the reason i went this way was I couldn't find anything directly exposing a way to go POJO from Offer object in there, as the DTO was only used on specific methods. tbh I was expecting a directly method on Offer object to convert to POJO, but yea definitely could be a better way to handle this in cleanup with reshuffle.

Reply:
> The general idea was that the Offer object is used internally, as you can read all the information you want from is, and the DTO (Data Transfer Object) is used for the API.

Ah gotcha! I can clean this up along with the reshuffle.